backend/src/main/java/de/neuefische/backend/api/dto/ApiResponseContent.java

```diff
@@ -0,0 +1,11 @@
+package de.neuefische.backend.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public record ApiResponseContent(
+        @JsonProperty("termine")
```

Comment:
Some as above: why do you choose "termine" when there is below "details"?

Reply:
It is a similar case to the other naming conventions from the external API, like the one above.